### DISCUSSION TITLE
Ollama model selectable in the Ollama Model Activity, but not in the Ollama Embeddings Activity

### DISCUSSION THREAD

**Discussion by greenmotion** [langflow-ai/langflow#5194] [General]
Just curious why models are selectable via a pulldown menu in the Ollama Model activity, but not in the Ollama Embeddings Activity?

Thank you,

  GM

**Comment by anovazzi1** [comment]
hey thanks for the feedback, i believe there is not a special reason for that, if you could create a github issue about it or create a pr updating the component we will be glad to merge it :)

**Comment by greenmotion** [comment]
Thank you for your response. I will create a github issue. Cheers.
